File: 2427-first-letter-to-appear-twice/2427-first-letter-to-appear-twice.cpp
class Solution {
public:
    char repeatedCharacter(string s) {
        unordered_map<char,int>mp; 
        for( auto c : s){
            if(mp.find(c) != mp.end()) return c; 
            mp[c]++;
        }
        return '\0'; 
    }
};

File: 2427-first-letter-to-appear-twice/2427-first-letter-to-appear-twice.java
class Solution {
    public char repeatedCharacter(String s) {
        Map<Character, Integer>mp = new HashMap<>(); 
        for( char c : s.toCharArray()){
            if(mp.containsKey(c)) return c; 
            mp.put(c,1); 
        }
        return '\0'; 
        
    }
}